src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceCharacterReference.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-row>
+    <v-col>
+      <Alert type="info">
+        <p>The Registry will not assess an application until references have been submitted. Please make sure your reference:</p>
+        <ul class="pl-5">
```

Comment:
this was needed due to a bug with vuetify and ol/ul not showing their bullets. 

https://github.com/vuetifyjs/vuetify/issues/17633

Reply:
This has been annoying me also

Reply:
Yeah pl-5 haha quick hack for now :P